src/tokens/fonts.js

```diff
@@ -1,32 +1,33 @@
 const fonts = {
   family: {
-    lato: 'Lato',
+    regular: `Lato-Regular`,
```

Comment:
this puts a constraint for us to add a new family. The ideal structure should be
```js
family: {
lato: {
regular:'Lato-Regular',
bold:'Lato-Bold',
}
}
```

so you can add n number of families

Reply:
Oh okay